ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!---
+There's currently no strict procedure or guidelines to submitting issues.
+Briefuly, just use common sense.
+
+Common sense at least includes:
+
+ * search for similar issues posted before creating a new issue.
+ * Use markdown to format all code/logs. Issues which are hard to read
+   when rendered on GitHub might be closed with a friendly reminder of this.
+
+Also, if applicacable add steps to reproduce a bug. And feel free to include sample code.
```

Comment:
`also, add steps to reproduce the bug, if applicable. Sample code would be nice too :)`